src/main/java/TaskList.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+public class TaskList {
+    ArrayList<Task> tasklist = new ArrayList<Task>();
+
+    public void listTasks() {
+        int i = 1;
+        for (Task task:tasklist) {
+            if (task.completed) {
+                System.out.println(" " + i + ".[X] " + task.taskName);
+            } else {
+                System.out.println(" " + i + ".[ ] " + task.taskName);
+            }
+            i++;
+        }
+    }
+
+    public void addTask(String taskName) {
+        tasklist.add(new Task(taskName));
+        System.out.println(" added: " + taskName);
+    }
+
+    public void markDone(int index) {
+        tasklist.get(index).completed = true;
```

Comment:
Accessing the inner fields of Task could be problematic if you change the implementation of Task later